chrome-launcher/chrome-launcher.ts

```diff
@@ -13,7 +13,7 @@ import {DEFAULT_FLAGS} from './flags';
 import {makeTmpDir, defaults, delay} from './utils';
 import * as net from 'net';
 const rimraf = require('rimraf');
-const log = require('lighthouse/lighthouse-core/lib/log');
+const log = require('lighthouse-logger');
```

Comment:
https://www.npmjs.com/org/lighthouse looks like it's already taken, but no one is using it...